src/broker/Manager.cc

```diff
@@ -681,7 +696,10 @@ bool Manager::PublishEvent(string topic, RecordVal* args)
 		xs.emplace_back(data_val->data);
 		}
 
-	return PublishEvent(std::move(topic), event_name, std::move(xs));
+	// At this point we come from script-land. This means that publishing of the event was
+	// explicitly triggered. Hence, the timestamp is set to the current network time. This also
+	// means that timestamping cannot be manipulated from script-land for now.
```

Comment:
Is that comment anymore special here in Broker than it is for `event test()` or `schedule 1min { test() };`? The timestamps of the above are just based on network_time and can't be manipulated either, otherwise I missed something somewhere.